README.md

```diff
@@ -1,5 +1,14 @@
 # Redis Labs helm repository
 
+## GKE quickstart
```

Comment:
Rename to 'GKE and helm initialization'?

Reply:
@kerneljake Please rename to Helm initialization on GKE,

Reply:
done